src/dotnet/Users.Contracts/UserStatus.cs

```diff
@@ -0,0 +1,9 @@
+namespace ActualChat.Users;
+
+public enum UserStatus
+{
+    None = 0,
```

Comment:
I would actually use `0` as `Active` (~ default user status in future / on DB upgrade / etc.) here, and use other values for other statuses.

Enum default should be meaningful unless it's `[Flags]` enum (where 0 = no flags), and I suspect `None` status is an error.

Reply:
done